dependencies/maven/rules.bzl

```diff
@@ -0,0 +1,261 @@
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
```

Comment:
We should make a comment at the top of every auto-generated file, to let the use know that this should not be edited. For example, see: https://github.com/graknlabs/grakn/blob/29b7c8a8e79b8e85b41cbf060b84d95adf1643d2/dependencies/maven/dependencies.bzl#L1